base-images/envd-starship/envd-starship.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu:20.04
+RUN apt-get update && apt-get install -y apt-utils
+RUN apt-get install -y --no-install-recommends --no-install-suggests --fix-missing curl ca-certificates
+RUN curl --proto '=https' --tlsv1.2 -sSf https://starship.rs/install.sh | sh -s -- -y
```

Comment:
Better to use multi-stage build here if only one binary is needed here. For example, https://github.com/tensorchord/Horust/blob/master/Dockerfile#L12

Reply:
make sense, PTAL